### DISCUSSION TITLE
Albums with various artists not being grouped in the library

### DISCUSSION THREAD

**Discussion by o-alquimista** [cmus/cmus#1389] [Troubleshooting]
Take the album "Need for Speed Underground 2", for example. On the filesystem, it's a single folder with `.mp3` files — all properly tagged with the same `album`.

![screenshot of file manager](https://github.com/user-attachments/assets/d8dfaec4-bd72-4ee5-8ac2-dbf2c0079c88)

Well, cmus literally pollutes the artist list with the many artists from that album — most of these contain only one song.

![screenshot of cmus](https://github.com/user-attachments/assets/2af6dc5f-760c-471e-af92-0c858e523dd9)

If I understand the manual correctly, it's supposed to be grouping them under `<Various Artists>`, but it's not.

> Note: Albums which feature various artists (e.g. samplers or
       compilations) are treated specially. If an album artist tag or the
       ID3v2 TPE2 frame is set, it will be used instead of the real artist
       name. Otherwise, cmus determines if the album is a compilation (if
       albumartist or artist are set to Various Artists, Various, VA, or V/A;
       or if compilation or partofacompilation are set to a truthy value; or
       if the ID3v2 TCMP frame is set). If so, the artist is named <Various
       Artists>.

![screenshot of ex falso tag editor](https://github.com/user-attachments/assets/17f0c490-5f66-4ab8-9615-19294f3c9336)

If the problem is a missing tag, then that's a big problem. You see, I get my music from torrents and then I keep seeding them — I can't change the files myself, unless I want to publish them as a new torrent.

VLC, for example, groups these albums under its own "Various Artists".

![Screenshot_20241214-150015_VLC](https://github.com/user-attachments/assets/5da0a3cc-fa32-4c2d-a1af-8a116e3f4050)

**Comment by gavtroy** [comment]
Usually the `Album Artist` tag would be set to "Various Artists". It's a common complaint so it would be nice if cmus could somehow work around it, but as it stands you're out of luck if you can't change the tags.

**Comment by seamusdemora** [comment]
Yes - `cmus` wants the metadata tagging done "the way it wants it done"  :)  I'm futzing around with that now.  Fortunately I've got the [`Mp3tag`](https://www.mp3tag.de/en/) app on my Mac to help with tag edits - it would be a nightmare using a CLI editor. And - not that I'm an expert (at all!) - `cmus` depends almost exclusively on tag values to organize and present the music library. This is different than some apps (e.g. in Sonos) that use only folder(s)/file(s) to organize the music library. 

Maybe (?) that could be a new/different mode of organization in `cmus`??  It might simplify things?? 

Once you start editing tags, don't forget to use the `update-cache` command to re-organize. I do this periodically as I make changes to avoid having to do a "giant re-do" of everything (w/ `update-cache -f`).
